Framework/Algorithms/src/MaxEnt.cpp

```diff
@@ -117,12 +120,24 @@ void MaxEnt::init() {
 
   auto mustBeNonNegative = boost::make_shared<BoundedValidator<double>>();
   mustBeNonNegative->setLower(1E-12);
-  declareProperty(make_unique<PropertyWithValue<double>>(
-                      "A", 0.4, mustBeNonNegative, Direction::Input),
-                  "A maximum entropy constant");
+  declareProperty(
+      make_unique<PropertyWithValue<double>>("A", 0.4, mustBeNonNegative,
+                                             Direction::Input),
+      "A maximum entropy constant. This algorithm was first developed for the "
+      "ISIS muon group where the default 0.4 was found to give good "
+      "reconstructions."
+      "In general the user will need to experiment with this value. Chosen a "
+      "small value may lead to unphysical spiky reconstructions and a too "
+      "large "
+      "value the reconstruction will start to resamble that of a direct "
+      "fourier "
+      "transform reconstruction. However, where data contain a strongly "
+      "determined zero "
+      "Fourier data point the reconstruction will be insensitive to the choice "
+      "of this property.");
```

Comment:
just a bit of cleaning up on the wording:

`In general the user will need to experiment with this value. Choosing a small value may lead to unphysical spiky reconstructions. Whereas, a large value will generate a reconstruction that resembles that of a direct Fourier transform reconstruction. However, when the data contains a strongly determined zero Fourier data point the reconstruction will be insensitive to the choice of this property.`